.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "isl"]
 	path = isl
 	url = https://github.com/ebi-gene-expression-group/isl.git
-	branch = develop_codon
+	branch = feature/fix_postprocessing
```

Comment:
This can be brought back to develop_codon, as feature/fix_postprocessing has been merged to develop_codon. Good to merge after this.